test/promise.js

```diff
@@ -152,4 +152,37 @@ describe('Promise', function () {
       }, 50);
     });
   });
-}); 
+  describe('Promise.prototype.then', function() {
+    var spy1 = sinon.spy();
```

Comment:
The spy code and SubClass need to be in a beforeEach. The spy is not reset on each test and the second test isn't testing anything. The first test calls it and the second test uses the same one